app/controllers/edit-service-name.controller.js

```diff
@@ -8,7 +8,7 @@ const formatPath = require('../utils/replace-params-in-path')
 const serviceService = require('../services/service.service')
 const { validateServiceName } = require('../utils/service-name-validation')
 
-exports.get = (req, res) => {
+exports.get = function getServiceName (req, res) {
```

Comment:
can we move the exports to the bottom of the file to to meet with our convention? https://pay-team-manual.cloudapps.digital/manual/coding-standards/node-js-style-guide.html#module-exports

Reply:
Done